.changeset/quick-shoes-visit.md

```diff
@@ -0,0 +1,5 @@
+---
+'@graphql-codegen/c-sharp-operations': patch
+---
+
+Use ampersand prefix for C# keyword name collisions
```

Comment:
```suggestion
Use at-sign (`@`) prefix for C# keyword name collisions
```

Reply:
Thanks, that is definitely not an ampersand!